.docker/gui-docker

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
should this file be ``.sh`` to make it clearer its a bash script? I'm not sure what the standard is but personally I always do it this way for clarity from a binary

Reply:
If anything it should be a .bash sh is different than bash. But it's standard for executables to not expose their underlying implementation in their name, only for setup files do you need the explicit extension since they're not cross compatible.  This can be executed from any shell.